py/selenium/webdriver/remote/webelement.py

```diff
@@ -47,7 +47,25 @@ class WebElement(object):
     reference is still valid.  This essentially determines whether or not the
     element is still attached to the DOM.  If this test fails, then an
     ``StaleElementReferenceException`` is thrown, and all future calls to this
-    instance will fail."""
+    instance will fail.
+
+    If an element is no longer part of the DOM a StaleElementReferenceException
+    will be thrown. An element is most often no longer part of the DOM because
+    the DOM has been destroyed. The DOM is destroyed when a new web page is loaded.
+    This even includes refreshing the same web page.
+
+    In order to handle a StaleElementReferenceException you must re-find the WebElement.
+
+    For Example:
+
+        try:
+            element = driver.find_element_by_css_selector(css_selector)
+            element.click()
+        except StaleElementReferenceException:
+            driver.get(url)
+            element = driver.find_element_by_css_selector(css_selector)
+            element.click()
```

Comment:
This example code is showing a very bad code practice. A more practical example would be using an expected_condition (ignoring stale element):

    from selenium.webdriver.common.by import By
    from selenium.webdriver.support.wait import WebDriverWait
    from selenium.webdriver.support.expected_conditions import visibility_of_element_located
    from selenium.common.exceptions import StaleElementReferenceException

    WebDriverWait(driver, 10, ignored_exceptions=StaleElementReferenceException)
        .until(visibility_of_element_located((By.CSS_SELECTOR, selector))).click()

(You don't actually have to supply the ignored_exceptions for visibility_of_element_located, the code actually ignores StaleElementReference... but I'm just trying to be verbose in showing how one uses expected conditions and can ignore this one)